extension/content/components/recipes/RecipeListing.js

```diff
@@ -2,7 +2,10 @@ import autobind from "autobind-decorator";
 import yaml from "js-yaml";
 import PropTypes from "prop-types";
 import React from "react";
-import { Button, Icon, Panel, Tag } from "rsuite";
+
+import { Link } from "react-router-dom";
+
+import { Button, Icon, Panel, Tag, ButtonToolbar } from "rsuite";
 import { convertToV1Recipe } from "devtools/utils/recipes";
```

Comment:
nit: add new line before this.

the rationale here is you are grouping library/framework imports and project imports